sideline-eglot.el

```diff
@@ -52,17 +60,39 @@
      ,@body
      (advice-remove 'y-or-n-p #'always)))
 
-(defvar-local sideline-eglot--ht-candidates nil
+(defvar sideline-eglot--ht-candidates nil
```

Comment:
Would this work with all buffers since this is no longer a local variable?

Reply:
also this is the only way to make this works with async json, because apparently it works on a temp buffer.